src/main/java/com/github/dockerjava/api/model/HostConfig.java

```diff
@@ -1,15 +1,21 @@
 package com.github.dockerjava.api.model;
 
-import javax.annotation.CheckForNull;
-
-import org.apache.commons.lang.builder.ToStringBuilder;
-
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.dockerjava.core.RemoteApiVersion;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+import javax.annotation.CheckForNull;
+import java.util.List;
 
+/**
+ * Used in `/containers/create`.
```

Comment:
In next PRs. That what i mentioned that reused object should be splitted. In this PR i only added fields as is.